fastlane/Fastfile

```diff
@@ -43,13 +43,15 @@ platform :ios do
     options[:configuration] ||= "Release"
     options[:include_bitcode] ||= false
     options[:export_method] ||= "enterprise"
+    options[:export_options] ||= { }
```

Comment:
Does passing in an empty object like this differ in what `gym` does when you don’t pass in `options` at all?

Reply:
It appears that fastlane will set it to an empty dictionary if no value was passed for that parameter. https://github.com/fastlane/fastlane/blob/master/fastlane/lib/fastlane/actions/build_ios_app.rb#L21